languages/en.ini

```diff
@@ -397,6 +397,8 @@ empty_search_disallowed = "An empty query is not allowed with the search target"
 Enable Auto Config = "Enable Auto Config"
 End Page = "End Page"
 eol_ellipsis = "…"
+epf_recommendations = "Publication Results"
```

Comment:
We use EPF specifically for *e-journal* title search, so we'll be re-labelling this in local/config to something like "Journal Results" or "eJournal Results".  But technically it's "publication" finder, so I figure that's what the default label should be.